ldap/src/main/java/pl/edu/icm/unity/ldap/endpoint/LdapApacheDSInterceptor.java

```diff
@@ -562,17 +616,24 @@ private void addAttribute(String name, String username, Collection<AttributeExt<
             toEntry.add(da);
         }
     }
-
-    /**
+    
+     /**
      * Get Unity user from LDAP search context.
      */
-    private EntryFilteringCursor getUnityUser(SearchOperationContext searchContext, String username)
+    private Entry getUnityUserEntry(SearchOperationContext searchContext, String username)
             throws LdapException
     {
         Entry entry = new DefaultEntry(lsf.getDs().getSchemaManager());
         try
         {
             long userEntityId = userMapper.resolveUser(username, realm.getName());
+            
+            Entity userEntity = identitiesMan.getEntity(new EntityParam(userEntityId));
+            for(Identity identity : userEntity.getIdentities()) {
```

Comment:
what's the purpose of this for?

Reply:
The identities were used during debugging, but have no function anymore. The for loop can be removed.